knife/knife.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.email = "adam@chef.io" # These seem a bit out of date, and this address probably doesn't go anywhere anymore?
   s.homepage = "https://www.chef.io"
 
-  s.required_ruby_version = ">= 2.7.0", "< 3.1"
+  s.required_ruby_version = ">3.0.0", "< 3.1"
```

Comment:
```suggestion
  s.required_ruby_version = ">= 3.0.0", "< 3.1"
```

Reply:
(I know technically `> 3.0.0` should be fine, but I have spun up something recently on `3.0.0` specifically so it seems an unnecessary boundary condition risk. Also... consistency with all the other things and adding a space after the `>=`

Reply:
agreed. done